website/blog/2017-04-13-1.0.0.md

```diff
@@ -0,0 +1,1062 @@
+---
+author: Christopher Chedeau (@vjeux)
+authorURL: https://twitter.com/vjeux
+title: Releasing Prettier 1.0
+---
+
+_This post was written by [vjeux](https://twitter.com/vjeux) and edited by
+[jlongster](https://twitter.com/jlongster), and originally published [here](http://jlongster.com/prettier-1.0)_
+
+We officially [announced](http://jlongster.com/A-Prettier-Formatter)
```

Comment:
Should this be copied over to the Prettier blog, too?

Reply:
Sounds good. Want to convert it to markdown?

Reply:
Should I include the live demo, or should I just make the examples static text?

Reply:
I forgot about that, not sure. Maybe doesn't make so much sense to import it then.